### PR TITLE
Add announcement for sunsetting PyPy

### PR DESCRIPTION
**Pull request by h-vetinari**
Draft for #2255, based on (my understanding of) the core call of today.

CC @conda-forge/core

### CODE REVIEW

**Comment by jaimergp** [comment]
The branch name 😂  The content has the information we need with a very gentle tone that I'm sure folks will appreciate. Thanks for that.

For long-ish announcements I recommend adding a first paragraph with the TLDR, then adding a `<!-- truncate -->` comment in its own line, and then we can write as much as we want. This way the `news/` page is quicker to look, and details can be obtained in the full, single-page view of the announcement.

**Comment by jaimergp** [comment]
LGTM. Tagging @mattip for visibility.

File: news/2024-08-10-sunsetting-pypy.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Sunsetting PyPy support
+
+TL;DR: We are planning to remove all PyPy builds in conda-forge in a few weeks,
```
Comment (mattip) [inline news/2024-08-10-sunsetting-pypy.md:3]: The original makes it sound like packages will be removed, I think the intention is to stop building new PyPy packages, correct?
```suggestion
TL;DR: We are planning to stop building new releases of packages for PyPy and
remove PyPy from conda-forge feedstock recipes in a few weeks,
```

**Comment by mattip** [comment]
Thanks also to the conda-forge team who put a lot of effort into making this work for as long as it did!

File: news/2024-08-10-sunsetting-pypy.md (hunk 1 above)
Comment (h-vetinari) [reply news/2024-08-10-sunsetting-pypy.md:3]: This is a good clarification, thanks. I'm just too packaging-brained, where "builds" are obviously new (because we don't ever remove artefacts).

**Review by jaimergp — APPROVED** [review]
